pandas/io/formats/style.py

```diff
@@ -508,7 +508,8 @@ def _apply(self, func, axis=0, subset=None, **kwargs):
         subset = _non_reducing_slice(subset)
         data = self.data.loc[subset]
         if axis is not None:
-            result = data.apply(func, axis=axis, **kwargs)
+            result = data.apply(func, axis=axis,
```

Comment:
@TomAugspurger I had to do this work-around here to push a list-like result into a frame (which we are breaking generally in this patch, but here style is relying on this, so kind of ok with this behavior, I have to add some more tests); this is from ``pandas/tests/io/formats/test_style.test_apply_axis``

. e.g.

```
In [5]:         df = pd.DataFrame({'A': [0, 0], 'B': [1, 1]})
   ...:         f = lambda x: ['val: {max}'.format(max=x.max()) for v in x]
   ...: 

In [6]: df
Out[6]: 
   A  B
0  0  1
1  0  1

In [7]: df.apply(f, axis=1)
Out[7]: 
0    [val: 1, val: 1]
1    [val: 1, val: 1]
dtype: object

In [8]: df.apply(f, axis=1, result_type='infer')
Out[8]: 
        A       B
0  val: 1  val: 1
1  val: 1  val: 1

```

I am not entirely sure why you need this this way.